### ISSUE TITLE
Command Blocks Could be used soon

### ISSUE THREAD

**Issue by tylerhyperHD** [TotalFreedom/TotalFreedomMod#682]
Command Blocks could now be available to admins only using PlayerInteractEvent and getting a clicked block type. You should be able to do the same way with minecart command blocks but with getting the item.

**Comment by JeromSar** [comment]
@tylerhyperHD Yes this is true, but it's not a stable solution. If only there was a CommandBlockSetCommandEvent...

**Comment by Wild1145** [comment]
@tylerhyperHD Could you give some additional information and respond to the other raised comments here please?
